gno.land/cmd/gnoland/show_validator.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gnolang/gno/tm2/pkg/amino"
+	"github.com/gnolang/gno/tm2/pkg/bft/privval"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+// Display a node's validator info.
+func newValidatorCmd(bc baseCfg) *commands.Command {
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			Name:       "validator",
```

Comment:
What if we merge the validator and node into a single command? This command will always display the node ID and optionally display validator information. We can also include additional information, such as the pruning strategy or any other relevant details that you would like to quickly check or share.